### PR TITLE
Toggle Trusted Button

### LINKED ISSUE #8551: NoteBooks: Trusted/Not Trusted button should toggle
Issue Type: <b>Feature Request</b>

Once you mark a notebook as trusted it cannot be "Not Trusted" . The button should toggle between trusted and not trusted state

Azure Data Studio version: azuredatastudio 1.13.1 (d08d07fe39c7fe74f9265db02cb963c38d048299, 2019-11-14T23:08:09.800Z)
OS version: Windows_NT x64 10.0.18363

### PR DESCRIPTION
**Pull request by rajeshka**
This PR fixes #8551
Trusted button now toggles between Not Trusted and Trusted

### CODE REVIEW

**Comment by chlafreniere** [comment]
Jupyter doesn't provide this ability (i.e. a notebook becomes not trusted again when it's renamed), so that's why we never did this in the first place.

Can you provide an example for why a user may want to do this?

**Comment by rajeshka** [comment]
What we found out that if we click Trusted by mistake there was no way to get back to not-trusted state.

File: src/sql/workbench/contrib/notebook/test/browser/notebookActions.test.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ suite('Notebook Actions', function (): void {
 		// Should stay trusted when trying to toggle again
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/test/browser/notebookActions.test.ts]: Update this comment as well

**Comment by rajeshka** [comment]
Trusted Button will toggle between Trusted and Not Trusted state

**Comment by Charles-Gagnon** [comment]
@chlafreniere Are you OK with this change?

**Comment by rajeshka** [comment]
Yes. It's been tested and test cases updated accordingly.

**Pull request merged**